libraries/botbuilder-ai/src/luis-client/luisClient.ts

```diff
@@ -51,11 +51,23 @@ export interface PredictionResolveOptionalParams {
     customHeaders: {headers: {[name: string]: string}};
 }
 
+ /**
+  *  Sets headers for request to LUIS service.
+  *  LUIS services use the `Ocp-Apim-Subscription-Key` header 
+  */
 export enum LuisApikeys {
     apiKeyHeader,
 }
 
+/**
+ * Luis Client Class
+ *
+ * @remarks
+ * This class is helper class for all the http request operations against the LUIS API
+ * https://www.luis.ai/
+ */
 export class LuisClient {
+    /** Acceptable values: https://westus.api.cognitive.microsoft.com */
```

Comment:
What should that URL retrieve?

Reply:
@Konigstiger Can we replace this comment with something like [this](https://github.com/Azure/azure-sdk-for-node/blob/master/lib/services/luis/runtime/lib/lUISRuntimeClient.d.ts#L23)(This is the comment of the former implementation of the _LuisClient_)

Reply:
It is just the allowed value of the _basePath variable. It is used internally by the constructor. We got feedback requesting that. We could only let the comment in the constructor. I will do that.